src/main/java/juja/sqlcmd/DatabaseManager.java

```diff
@@ -40,6 +40,19 @@ public boolean insert(String tableName, DataSet dataset) {
         }
     }
 
+    /*
+     * Method for deleting data from a table.
+     * If successful, the method returns true, else return false
+     */
+    public boolean delete(String tableName, int id) {
+        try (Statement statement = connection.createStatement()) {
+            return statement.executeUpdate(String.format("DELETE FROM %s WHERE id = %s", tableName, id)) > 0;
```

Comment:
Extracting of 'String.format("DELETE FROM %s WHERE id = %s", tableName, id)' to separate variable will bring more readability.